.github/workflows/translators-sync.yml

```
@@ -0,0 +1,46 @@
+name: Sync Translators Branch
+
+permissions:
+  actions: write
+  checks: write
+  contents: write
+  deployments: write
+  pull-requests: write
+  statuses: write
+  
+on:
+  push:
+    branches:
+        - 'master*'
+        
+jobs:
+    sync-translation:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout 🛎️
+              uses: actions/checkout@v2.3.1
```

```suggestion
              uses: actions/checkout@v4.1.1
```

Latest version is v4.1.1
https://github.com/actions/checkout